apex/contrib/optimizers/distributed_fused_lamb.py

```diff
@@ -266,13 +256,60 @@ def allreduce_hook(*unused):
         self._total_param_size = p_offset
         dwu_min_page_size = 256 * self._num_blocks * self._num_chunks * self._group_size
         self._total_param_size = ((self._total_param_size + dwu_min_page_size - 1) // dwu_min_page_size) * dwu_min_page_size
+        self._new_params = torch.zeros([self._total_param_size], dtype=torch.uint8 if self._e5m2_allgather else torch.float16, device='cuda')
+
+
+
+    def _lazy_init_stage1(self):
+        if self._lazy_init_stage1_done: return
+
+        p_offset = 0
+        p_i = 0
+        #self._model_params = []
+        #self._grad_accs = []
+        #self._group_properties = []
+        for group in self.param_groups:
+            prev = None
```

Comment:
prev, beta, bias_correction, eps, weight_decay, p_offset not used and needed in this for loop

Reply:
thanks! Just removed those.